### DISCUSSION TITLE
Battery Charge %

### DISCUSSION THREAD

**Discussion by Redgewell** [StealthChesnut/HA-FoxESS-Modbus#117] [General]
I have LAN integration working with my HA setup and its fabulous. via Fox app/cloud I can see a battery cahage %, eg 45% charged. Can I get (or calculate) this via the Modbus integration? (I have 3 x FOX Mira 2.5kw HV25)

**Comment by Daibutt** [comment]
This should be your battery % charge;
![image](https://user-images.githubusercontent.com/69276073/228573733-a2bae4cb-7836-42bb-bd1d-6a72350cd082.png)

**Comment by Redgewell** [comment]
yep, <stupid user. was the only sensor i didn't add to my card for some reason and thought SoC was something to do with the max allowed discharge. / lithium protection.. guessing its "stage of charge" ? thanks for quick response @Daibutt

**Comment by Daibutt** [comment]
Yep,  State of Charge
